### ISSUE TITLE
[Request] init/add should have an option -force

### ISSUE THREAD

**Issue by Rand-Random** [NuGet/Home#10618]
**NuGet.exe**

It would be nice if NuGet.exe could introduce an optional argument for init/add to force the packages, instead of bringing this warning
![image](https://user-images.githubusercontent.com/9868626/109931809-9da46700-7cc9-11eb-994a-b3d8f6ddef3c.png)

This would allow me to update my local source more easily, currently I have to delete those files manually before I can call init/add.

**Comment by george-petrakis** [comment]
This should have been implemented already, it's an issue in github actions where the source has already been added.

**Comment by SimonDarksideJ** [comment]
Yeah, really frustrating, a tester inadvertently pushed a pre-release version of our packages and we cannot override them with the actual release now.
I requested to deprecate but no update all day on that.
